libs/libpcf/src/io/pcf_reader.cpp

```diff
@@ -57,12 +57,17 @@ int read_pcf(const char* fname, PcfData& pcf_data) {
           pcf_data.set_io_pin(io_id, pin_name);
         } else if (word[0] == COMMENT) {  // if it's a comment
           break;  // or ignore the full line comment and move on
+        } else if (word.find("set_clk") == 0 || word.find("set_reset") == 0) {
```

Comment:
My suggestion is to add a new option ``--reduce_error_to_warning`` to the read_pcf command.